scripts/index.js

```diff
@@ -141,6 +143,7 @@ fs.emptyDir(paths.appBuild)
         }
 
         spinner.succeed();
+        runHook(afterRebuildHook);
```

Comment:
I think it'd be better to use `ora` to display a message before running the hooks so the user knows what is happening.